lib/mintel/alerts/PrometheusRule-kube-operators-redis.rules.yaml

```diff
@@ -0,0 +1,111 @@
+---
+apiVersion: monitoring.coreos.com/v1
```

Comment:
@nickmintel why is this file added as yaml ?

Reply:
My bad, unwanted debug file (now removed)